recipes/perl-encode/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "perl-encode" %}
+{% set version = "3.01" %}
+{% set sha256 = "d4555f570491648dbbd602bce7966672834b4c8f45acaa6757de474fca3a4d87" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: Encode-{{ version }}.tar.gz
+  url: https://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Encode-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - perl >=5
+  run:
+    - perl >=5
```

Comment:
You need to pin this to `x.x.x` as the package install path has the directory version